tsconfig.json

```diff
@@ -26,6 +26,6 @@
     "baseUrl": "./src/",
     "outDir": "./build"
   },
-  "include": ["src/**/*.ts", "test/**/*.ts", "node_modules/web3-*/types/*.d.ts"],
+  "include": ["src/**/*.ts", "test/**/*.ts", "node_modules/web3-*/types/*.d.ts", "node_modules/moment/moment.d.ts"],
```

Comment:
moment types aren't picked up automatically?

Reply:
they weren't for me, now we can reevaluate in the refactor of dex-js